server/conf/evolutions/default/57.sql

```diff
@@ -0,0 +1,24 @@
+-- These assume a default locale of en_US, which is true for all deployments as of 2023-09-05.
+
+# --- !Ups
+UPDATE ONLY questions
+SET question_options = (
+  SELECT jsonb_agg(
+           jsonb_set(
+             question_option,
+             '{adminName}',
+             to_jsonb(question_option->'localizedOptionText'->'translations'->'en_US'),
+             true
+             )
+           )
+  FROM jsonb_array_elements(question_options) AS question_option
+)
+WHERE question_options IS NOT NULL;
+
+# --- !Downs
+UPDATE ONLY questions
+SET question_options = (
+  SELECT jsonb_agg((question_option - 'adminName'))
+  FROM jsonb_array_elements(question_options) AS question_option
+)
+WHERE question_options IS NOT NULL;
```

Comment:
**suggestion (if-minor):** if this ends up being too much of a mess then don't bother, but consider writing a test for the up case here. I'm curious what happens when:

- [ ] question_options is empty
- [ ] question_options is missing an `en_US` translation
- [ ] the `en_US` translation is an empty string

The `Evolutions` class will be helpful for writing this test (see https://stackoverflow.com/questions/46984028/how-to-apply-play-evolutions-when-running-tests-in-play-framework)

Reply:
I put a few hours into this and didn't get it working, so I tested these edge cases manually and updated the query to handle them.

I wrote #5559 to capture this goal, though.

Reply:
Thank you for making an effort at it!